builder/builder-source-archive.c

```diff
@@ -426,6 +427,24 @@ unrpm (GFile   *dir,
   return res;
 }
 
+static gboolean
+undeb (GFile   *dir,
+       const char *deb_path,
+       GError **error)
+{
+  gboolean res;
+  const gchar *argv[] = { "sh", "-c", NULL, NULL };
+  char *undeb_cmdline = g_strdup_printf("ar t %s | grep data.tar | xargs ar x %s",
```

Comment:
Yeah I did this to keep dependencies low as ar & grep are almost always installed, not necessarily dpkg-deb, but yeah why not lets change it, it's a bit simpler than hard-coding this piped around thing :) Will fix shell injection bug.